domene/src/main/kotlin/no/nav/tiltakspenger/domene/vedtak/Vedtak.kt

```diff
@@ -14,9 +15,9 @@ data class Vedtak(
     val behandling: BehandlingIverksatt,
     val vedtaksdato: LocalDateTime,
     val vedtaksType: VedtaksType,
-    // de under kan hentes fra behandling men kanskje kjekt å ha her også?
     val periode: Periode,
     val saksopplysninger: List<Saksopplysning>,
+    val utfallsperioder: List<Utfallsperiode>,
```

Comment:
Det er veldig mye av de samme dataene som ligger under både Vedtak og Behandling, hva er begrunnelsen for det? Kunne det vært like greit å lagt Vedtak under Behandling, med alle de tilhørende dataene?

Da kunne man flyttet en del av status-greiene inn i vedtak også kanskje? Det er vel vedtaket som er innvilget eller avslått, ikke behandlingen? Behandlingen er bare ferdigbehandlet.. :)

Reply:
Mulig dette ikke hører inn under denne PRen altså..